packages/app/obojobo-express/server/viewer/viewer_notification_state.js

```diff
@@ -0,0 +1,57 @@
+const db = oboRequire('server/db')
+
+function getStatus(id) {
+	return db.oneOrNone(
+		`
+		SELECT status FROM notification_status
+		WHERE
+			id = $[id]
+	`,
+		{
+			id
+		}
+	)
+}
+function getTitle(id) {
+	return db.oneOrNone(
+		`
+		SELECT title FROM notification_status
+		WHERE
+			id = $[id]
+	`,
+		{
+			id
+		}
+	)
+}
+function getText(id) {
+	return db.oneOrNone(
+		`
+		SELECT text FROM notification_status
+		WHERE
+			id = $[id]
+	`,
+		{
+			id
+		}
+	)
+}
+function getId(lastLoginDate) {
+	return db.oneOrNone(
+		`
+			SELECT id FROM notification_status
+			WHERE created_at >= $[lastLoginDate]
+			LIMIT 1
+		`,
+		{
+			lastLoginDate
+		}
+	)
+}
```

Comment:
Several things here:

- Change the name from `getId` to something more descriptive like `getRecentNotifications`.
- Generalize `lastLoginDate` to `targetDate` or maybe even just `date`.
- Use `manyOrNone` instead of `oneOrNone` and remove the `LIMIT` so this function grabs all notifications after a given date.
- Add an `ORDER BY created_at ASC` so notifications always appear in a predictable order.